reccmp/isledecomp/compare/core.py

```diff
@@ -24,6 +24,9 @@
 logger = logging.getLogger(__name__)
 
 
+NamedOrigEntry = TypedDict("NamedOrigEntry", {"orig": int, "name": str})
```

Comment:
Not too happy with this solution since this type is only used internally in one function. There's likely a better solution.